docs/README.md

```diff
@@ -22,6 +22,10 @@ You can execute it as part of your development process as an Eclipse IDE plugin
 * [jSparrow Maven plugin](maven/getting-started.html)
 * [jSparrow GitHub App](github/getting-started.html)
 
+## jSparrow 3.2.0 has been released!
+
+We are now supporting Eclipse versions using Java 11 and Java 11 projects! See [Release Notes](/eclipse/release-notes.html#_3-2-0). 
+
 ## jSparrow 3.1.0 has been released!

```

Comment:
IMHO we should only have the latest release on the front page. I removed release 3.0.0 last time.